test/midje/emission/plugins/t_util.clj

```diff
@@ -15,6 +15,8 @@
   ;; Note ordering
   (attractively-stringified-value {:b 2 :a 1}) => "{:a 1, :b 2}"
   (attractively-stringified-value #{9 6 2 7 1 3}) => "#{1 2 3 6 7 9}"
+  (read-string (attractively-stringified-value #{[1] [:a]})) => #{[1] [:a]}
+  (read-string (attractively-stringified-value {[1] "1" [:a] "a"})) => {[1] "1" [:a] "a"}
```

Comment:
A tricky way to test this data-to-string function without relying on the ordering of the data.
I wasn't able to create a test at the `fact` level because I wasn't able to get `silent-fact`/`note-that` to work in my favor in this context..